opennms-base-assembly/src/main/filtered/etc/eventd-configuration.xml

```diff
@@ -1,2 +1,2 @@
-<EventdConfiguration xmlns="http://xmlns.opennms.org/xsd/config/eventd" TCPAddress="127.0.0.1" TCPPort="5817" UDPAddress="127.0.0.1" UDPPort="5817" receivers="5" getNextEventID="SELECT nextval('eventsNxtId')" socketSoTimeoutRequired="yes" socketSoTimeoutPeriod="3000">
+<EventdConfiguration xmlns="http://xmlns.opennms.org/xsd/config/eventd" TCPAddress="127.0.0.1" TCPPort="5817" UDPAddress="127.0.0.1" UDPPort="5817" receivers="5" getNextEventID="SELECT nextval('eventsNxtId')" socketSoTimeoutRequired="yes" socketSoTimeoutPeriod="3000" threads="300">
```

Comment:
Can we avoid modifying this file? Other sink modules default to (2*number of cores) for the number of threads when none are set.

Reply:
Yes thats the main what we have, this field is like configurable like how syslogd-configuration.xml and trapd-configuration.xml has where we can even make threads configurable.